pipeline-modules/deployment-service/gcp/private-helm.yaml

```diff
@@ -107,7 +112,7 @@ template: |
       kubectl create ns {{namespace}}
       {% endif %}
       cd {{source_code_path}}/{{helm_chart_path}} ;
-      helm upgrade --install -f {{source_code_path}}/{{helm_values_path}} {{applicationName}}-{{environment}} . --atomic --timeout {{deployment_timeout}} --namespace {{namespace}}
+      helm upgrade --install -f {{source_code_path}}/{{helm_values_path}} {{applicationName}}-{{environment}} . --atomic --timeout {{deployment_timeout}} --namespace {{namespace}} {% if helm_values_set_var %} --set {{helm_values_set_var}}  {% endif %}
```

Comment:
are you sure you can do this:
helm --set key1=val1,key2=val2
vs
helm --set key1=va1 --set key2=val2
i.e. have you tested this with more than on key in "helm_values_set_var"?

Reply:
I use this command
helm install neo4j-db . -n neo4j-db --set neo4j.password='XXXXXXX',fullnameOverride=neo4j-db 

It worked for me